### ISSUE TITLE
fix parenthesis error in deprecation docs

### ISSUE THREAD

**Issue by brainwane** [pypa/warehouse#8031]
I'd like for @Manthan03583 to take this one.

https://warehouse.readthedocs.io/application/#historical-context-deprecations has a grammar error in the line

> download counts visible in the API: instead, use the Google BigQuery service)

This is a good first contribution for someone who's new to this repository. You open up `docs/application.rst` and change

~~~
- `download counts visible in the API <https://warehouse.readthedocs.io/api-reference/xml-rpc/#changes-to-legacy-api>`_:
  instead, use `the Google BigQuery service <https://packaging.python.org/guides/analyzing-pypi-package-downloads/>`_)
~~~

to

~~~
- `download counts visible in the API <https://warehouse.readthedocs.io/api-reference/xml-rpc/#changes-to-legacy-api>`_
  (instead, use `the Google BigQuery service <https://packaging.python.org/guides/analyzing-pypi-package-downloads/>`_)
~~~

and then open a pull request.

--

**Good First Issue** for @Manthan03583: For directions for getting set up, see our [Getting Started Guide](https://warehouse.pypa.io/development/getting-started/). If you are working on this issue and have questions, feel free to ask them here, [`#pypa-dev` on Freenode](https://webchat.freenode.net/?channels=%23pypa-dev), or the [distutils-sig.python.org mailing list](https://mail.python.org/mailman3/lists/distutils-sig.python.org/).

**Comment by Manthan03583** [comment]
Thank you

**Comment by Manthan03583** [comment]
mailing list link in developer docs updated #8030

**Comment by brainwane** [comment]
Hi @Manthan03583 - please come into IRC https://webchat.freenode.net/?channels=%23pypa-dev now, if you can.

**Comment by Manthan03583** [comment]
sorry mam , my internet connection is running slowly now. And I am sorry i mixed both the commits in one pull request.

**Comment by brainwane** [comment]
@Manthan03583 

I think you need some basic help using git before you continue. Instead of committing your proposed changes to your `master` branch, you need to create a new branch and commit the changes to that new branch. I suggest that you read a training kit from

https://github.github.com/training-kit/ (available in several languages)

from start to finish, and that you read 

https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/proposing-changes-to-your-work-with-pull-requests

as well. This will probably take you a few days. If you need some more guidance to understand how to make a new branch and create a pull request from it, please take a look at https://zulip.readthedocs.io/en/latest/git/ which may also help you.

Then, after you've done that, please open a new PR in this repository, one that just contains https://github.com/pypa/warehouse/pull/8030/commits/bbeb7d11e9cf4ddd6770d91f0d48ae144d47fa2c . 

Thanks.

**Comment by Manthan03583** [comment]
parenthesis error in deprecation docs fixed #8046
